### PR TITLE
Component/4087 toast

### PR DESCRIPTION
**Pull request by Cal-L**
**Description**

#4087 

Account variant
image

Network variant
image

Plain variant
image

**Screenshots/Recordings**

_If applicable, add screenshots and/or recordings to visualize the before and after of your change_

**Issue**

Progresses #???

**Checklist**

* [ ] There is a related GitHub issue
* [ ] Tests are included if applicable
* [ ] Any added code is fully documented

### CODE REVIEW

File: app/component-library/components/Toast/Toast.styles.ts [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * @returns StyleSheet object.
+ */
+const styleSheet = StyleSheet.create({
+  base: {
+    position: 'absolute',
+    width: toastWidth,
+    left: marginWidth,
+    bottom: 0,
+    backgroundColor: colors.background.alternative,
```
Comment (jpcloureiro) [inline app/component-library/components/Toast/Toast.styles.ts:20]: These colors won't follow app theme. Is that the desired behavior?

File: app/component-library/components/Toast/Toast.tsx [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+      <Link onPress={linkOption.onPress} variant={BaseTextVariant.sBodyMD}>
+        {linkOption.label}
+      </Link>
+    ) : null;
+
+  const renderToastContent = (options: ToastOptions) => {
+    let avatarContent: React.ReactNode = null;
+    const { labelOptions, linkOption } = options;
+
+    switch (toastOptions?.variant) {
```
Comment (jpcloureiro) [inline app/component-library/components/Toast/Toast.tsx:137]: Suggestion: Move the switch logic inside a funcion that deals with the avatar rendering. We will have better separation of concerns.

```
  const renderAvatar = (toastOpts: ToastOptions) => {
    

    const { variant } = toastOpts;

    switch (variant) {
      case ToastVariant.Network: {
        const { networkImageUrl } = toastOpts;

        return (
          <NetworkAvatar
            networkImageUrl={networkImageUrl}
            size={BaseAvatarSize.Md}
            style={styles.avatar}
          />
        );
      }
      case ToastVariant.Account: {
        const { accountAddress } = toastOpts;
        return (
          <AccountAvatar
            accountAddress={accountAddress}
            type={AccountAvatarType.JazzIcon}
            size={BaseAvatarSize.Md}
            style={styles.avatar}
          />
        );
      }
      case ToastVariant.Plain:
      default:
        return null;
    }
  };
```

File: app/component-library/components/Toast/Toast.styles.ts (hunk 1 above)
Comment (Cal-L) [reply app/component-library/components/Toast/Toast.styles.ts:20]: Yep it is, we're locking Toast to use dark theme colors

File: app/component-library/components/Toast/Toast.tsx (hunk 2 above)
Comment (Cal-L) [reply app/component-library/components/Toast/Toast.tsx:117]: Good call. Updated!

**Review by jpcloureiro — APPROVED** [review]
LGTM!

**Pull request merged**